File: archive/samples/python/53.teams-messaging-extensions-action-preview/example_data.py
# Copyright (c) Microsoft Corporation. All rights reserved.
# Licensed under the MIT License.


class ExampleData(object):
    def __init__(
        self,
        question: str = None,
        is_multi_select: bool = False,
        option1: str = None,
        option2: str = None,
        option3: str = None,
    ):
        self.question = question
        self.is_multi_select = is_multi_select
        self.option1 = option1
        self.option2 = option2
        self.option3 = option3

File: archive/samples/csharp_dotnetcore/53.teams-messaging-extensions-action-preview/ExampleData.cs
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

namespace Microsoft.BotBuilderSamples
{
    public class ExampleData
    {
        public ExampleData()
        {
            MultiSelect = "true";
            UserAttributionSelect = "true";
        }

        public string SubmitLocation { get; set; }

        public string Question { get; set; }

        public string MultiSelect { get; set; }

        public string Option1 { get; set; }

        public string Option2 { get; set; }

        public string Option3 { get; set; }

        public string UserAttributionSelect { get; set; }
    }
}
